lib/dor/services/client/object_version.rb

```diff
@@ -95,6 +112,17 @@ def inventory
           JSON.parse(resp.body).fetch('versions').map { |params| Version.new(**params.symbolize_keys!) }
         end
 
+        # @return [VersionStatus] status of the version
+        # @raise [UnexpectedResponse] on an unsuccessful response from the server
+        def status
+          resp = connection.get do |req|
+            req.url "#{base_path}/status"
+          end
+          raise_exception_based_on_response!(resp, object_identifier) unless resp.success?
+
+          VersionStatus.new(JSON.parse(resp.body).symbolize_keys!)
+        end
```

Comment:
Can you add a line or two of documentation roughly [here-ish](https://github.com/sul-dlss/dor-services-client/blame/main/README.md#L121)? It might be good to call out that the return value is a struct with certain fields in the docs too.

Reply:
Yes. Done.